heliopy/data/util.py

```diff
@@ -176,6 +181,40 @@ def units_attach(data, units):
     return timeseries_data
 
 
+def cdf_units(cdf_, keys=None):
+    """
+    Takes the CDF File and the required keys, and finds the units of the
+    selected keys.
+
+    Parameters
+    ----------
+    cdf_ : cdf
+        Opened cdf file
+    keys : :class 'dict'
+        The keys required by user.
```

Comment:
Need to explain here that `keys` is optional, and what happens when `keys=None`.